ex02.exs

```diff
@@ -1,6 +1,32 @@
 
 defmodule Ex02 do
+  def new_counter(value \\ 0) do
+      {:ok, counter} = Agent.start_link(Counter, :new, [value])
+      counter
+  end
+
+  def next_value(counter) do
+    Agent.get_and_update(counter, Counter, :next_value, [])
+  end
+
+  @me TheCounter
+  def new_global_counter(value \\ 0) do
+    {:ok, counter} = Agent.start_link(Counter, :new, [value], name: @me)
+    counter
+  end
+
+  def global_next_value() do
```

Comment:
Just a question: why is this one `&Counter.next_value/1`, but line 14 uses the MFA form?